.github/workflows/website-root.yml

```diff
@@ -0,0 +1,109 @@
+name: Azure Static Web App Root
+
```

Comment:
Do we normally change this file?

Reply:
yes, we have to create it bc it doesn't exist while in preview.